scopes/code/ui/code-compare/code-compare.module.scss

```diff
@@ -0,0 +1,44 @@
+@import '~@teambit/ui-foundation.ui.constants.z-indexes/z-indexes.module.scss';
+$panelBg: #fafafa;
+
+.componentCompareCodeContainer {
+  display: flex;
+  height: 100%;
+  width: 100%;
+}
+
+.left {
+  min-width: 200px;
+  //cannot assign max-width directly to right side
+  max-width: calc(100% - 200px);
+  display: flex;
+}
+
+.right {
+  position: relative;
+  background: $panelBg;
```

Comment:
we should have a token for this

Reply:
we dont, all pages are defining this variable